eduNEXT/xblock-imagesgallery
# feat: button save images edit xblock

### Description

This feature allows you to upload and delete images after clicking the **'Save'** button and see a preview of the gallery


![save-button-demo](https://github.com/eduNEXT/xblock-imagesgallery/assets/134975835/eb19fb3b-9ac6-40e2-865c-9653128873c2)



### How to test it

- Check this branch`jv/edit-xblock-save-button` in the XBlock images gallery
1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 
```


**Merge checklist:**
Check off if complete *or* not applicable:
